src/utils/githubApi.ts

```diff
@@ -31,22 +31,9 @@ export class GitHubRemote {
   ) {
     this.owner = owner;
     this.repo = repo;
-    if (username && apiToken) {
-      this.setAuth(username, apiToken);
-    }
```

Comment:
should be able to remove these parameters entirely -- or maybe take the api key and do the header modification here?